package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "postinstall": "lerna bootstrap",
     "build": "lerna run compile",
-    "lint": "eslint . --ext .ts --ext .js",
+    "lint": "eslint . --ext .ts --ext .js && lerna run lint",
```

Comment:
eslint should cover all packages.

Reply:
> Since the front-end web style can be quite different from node.js style, I just add separate lint rules to pipcook-board-client (just use standard Alibaba Ice rules)

So it's trying to add multiple lint rules, but this seems adding duplicated checks, how about  a scope way, like: `lerna run --scope @pipcook/pipcook-board-client lint`?

Reply:
And we should exclude the board-client source from https://github.com/alibaba/pipcook/blob/master/.eslintrc.js :)

Reply:
I exclude the board-client from .eslintignore. `lerna run lint` will not cause duplicate checks since only pipcook-board-client has lint command. Anyway, it makes sense to add scope to avoid any unexpected behaviour later.